usmqe/api/tendrlapi/common.py

```diff
@@ -194,3 +196,60 @@ def flows(self, asserts_in=None):
         self.check_response(response, asserts_in)
         # TODO: some minimal validation of flows response?
         return response.json()
+
+    def get_nodes(self):
+        """ Get list node ids.
+
+        Name:        "get_nodes",
+        Method:      "GET",
+        Pattern:     "GetNodeList",
+        """
+        pattern = "GetNodeList"
+        response = requests.get(
+            pytest.config.getini("usm_api_url") + pattern,
+            auth=self._auth)
+        self.print_req_info(response)
+        self.check_response(response)
+        return response.json()
+
+    def import_cluster(self, nodes, sds_type=None):
+        """ Import cluster.
+
+        Name:        "import_cluster",
+        Method:      "POST",
+        Pattern:     "ImportCluster",
+
+        Args:
+            sds_type: ceph or glusterfs
+            nodes: node list of cluster which will be imported
+        """
+        pattern = "ImportCluster"
+        response = requests.post(
+            pytest.config.getini("usm_api_url") + pattern,
+            data=json.dumps({
+                "sds_type": sds_type,
```

Comment:
I am afraid you have to put an `if` here, because if `sds_type` is not used, we should not send even `None`.